CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# How To Contribute
+
+## Installation
+
+- `git clone https://github.com/ember-cli/ember-exam.git`
+- `cd ember-exam`
+- `yarn install`
+
+## Linting
+
+- `yarn lint:hbs`
+- `yarn lint:js`
+- `yarn lint:js --fix`
+
+## Running tests
+
+- `ember test` – Runs the test suite on the current Ember version
+- `ember test --server` – Runs the test suite in "watch mode"
+- `ember try:each` – Runs the test suite against multiple Ember versions
```

Comment:
we should add `yarn test:node` to run tests from `/node-test`
and replace `ember try:each` with `yarn test:all`

Reply:
Done!